llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -404,6 +405,138 @@ Value *VPInstruction::generateInstruction(VPTransformState &State,
     Builder.GetInsertBlock()->getTerminator()->eraseFromParent();
     return CondBr;
   }
+  case VPInstruction::ComputeReductionResult: {
+    if (Part != 0)
+      return State.get(
+          this, VPIteration(State.UF - 1, VPLane::getLastLaneForVF(State.VF)));
```

Comment:
This VPInstruction generates a single scalar value, rather than per-part/per-lane. Suffice to store it in State under Part 0 Lane 0, as in other uniform values?